### PR TITLE
Añadir documentacion de rutas virtuales

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

> Añadir documentacion de rutas virtuales

### Tasks:

- [x] Añadir imagenes
- [x] Traducir textos

### Links:

* [Crear rutes virtuals](http://builds.gisce.net/powerp-docs/powerp_imp_gis_rotues/ca/gis/webgis/#crear-rutes-virtuals)
* [Crear rutas virtuales](http://builds.gisce.net/powerp-docs/powerp_imp_gis_rotues/es/gis/webgis/#crear-rutas-virtuales)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_gis_rotues/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_gis_rotues/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/gis/webgis.md [hunk 1]
```diff
@@ -127,6 +127,20 @@
 
  ![](_static/coordenades.png)
 
+## Crear rutes virtuals
+
+1. Exportar els comptadors del lot
+
+2. Activar la capa de  Rutes\No comunica
+
+3. Premer el control de dibuix de la ruta i dibuixar l'area ruta(1) ![](_static/rutes_control.png)
```
Comment (gdalmau) [inline docs/gis/webgis.md]: `Premer` → `Prémer`
`area` → `àrea`

(es) `Pular` → `Pulsar`
(es) `area` → `área`

File: docs/gis/webgis.md [hunk 2]
```diff
@@ -127,6 +127,20 @@
...
+
+4. Si es vol es pot modificar l'area de l'area de la ruta amb el control(2)  ![](_static/rutes_control.png)
+
+5. Enviar la ruta usant el control ![](_static/control_send_route.png)
+
+6. S'obrirà un avis informant de que la ruta s'ha creat i es baixara el fitxer de ruta
```
Comment (gdalmau) [inline docs/gis/webgis.md]: `avis` → `avís`
`baixara` → `baixarà`

(es) `abrira` → `abrirá`
(es) `bajara` → `bajará`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_gis_rotues/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**